app/lib/utils/translations/translations_transaction.dart

```diff
@@ -40,6 +53,36 @@ class TranslationsEnTransaction implements TranslationsTransaction {
       'Your registration for the next cycle has been cancelled. If you change your mind, you can register again.';
   @override
   String get unregisterParticipantNotificationTitle => 'Registration cancelled';
+  @override
+  String get alreadyEndorsedErrorBody => 'This account has already been endorsed for this cycle.';
+  @override
+  String get alreadyEndorsedErrorTitle => 'Already Endorsed';
+  @override
+  String get noValidClaimsErrorBody => 'You did not send any valid claims. Did you scan the other attendees?';
+  @override
+  String get noValidClaimsErrorTitle => 'No Valid Claims';
+  @override
+  String get votesNotDependableErrorBody =>
+      "Only half or less of the assigned participants were attested for this gathering. It could also be that some attendees haven't submitted their attestation yet. This prevents the early payout, and you need to wait for 48 hours.";
+  @override
+  String get votesNotDependableErrorTitle => 'Votes not dependable';
+  @override
+  String get insufficientFundsErrorBody =>
+      'You do not have sufficient funds on this account. See on the website of your local community how to get some.';
+  @override
+  String get insufficientFundsErrorTitle => 'Insufficient Funds';
+  @override
+  String get txTooLowPriorityErrorBody =>
+      'Technical transaction priority error. This can happen if you tap twice on a submit button very quickly. Please wait for a few seconds.';
+  @override
+  String get txTooLowPriorityErrorTitle => 'Transaction priority error';
+  @override
+  String get rewardsAlreadyIssuedErrorBody =>
+      'Another attendee has triggered the payout for this gathering. You should have received your income already.';
+  @override
+  String get rewardsAlreadyIssuedErrorTitle => 'Rewards already issued';
+  @override
+  String get invalidTransactions => 'Invalid Transactions';
 }
```

Comment:
As I said above, transaction error fits better.

```suggestion
  @override
  String get transactionError => 'Transaction error';
}
```